CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to RunDMCMC
+
+This is a guide to contributing to the RunDMCMC project. This guide is
+currently (as of 2018-06-11) in an early form and may change rapidly. These are
+not strict rules; break them when appropriate. Feel free to make a pull request
+modifying this document.
+
+## How can I contribute?
+
+### Reporting bugs
+
+If you find a bug while using any tools in this repository, consider creating a
+[GitHub issue](https://github.com/gerrymandr/RunDMCMC/issues) documenting the
+problem. Be sure to be detailed. Explain what you did, what you expected to
+happen, and what actually happened.
+
+### Updating documentation
+
+If you find any documentation that is lacking or in error, submit a pull
+request that updates it. Be sure to follow the [documentation
+styleguide](#documentation-styleguide).
+
+### Suggesting enhancements
+
+If you have ideas for additional features, consider creating a [GitHub
+issue](https://github.com/gerrymandr/RunDMCMC/issues) documenting the desired
+feature. Be sure to be detailed. Explain what the tool currently does (or does
+not do), what you would like it to do, and how this benefits users.
+
+### Contributing code (pull requests)
+
+The main way to contribute code to the project is via GitHub pull requests
+(PRs). To create a pull request, see the [GitHub
+docs](https://help.github.com/articles/creating-a-pull-request/). Be detailed
+in the description of your pull request. Explain what your proposed changes do,
+and reference any relevant issues.
+
+Be sure to follow the [style guides](#style-guides) for code and documentation.
+
+## Style Guides
+
+### Git commit messages
+
+When writing git commit messages, use the following guidelines:
+
+- Use the present tense ("Update documentation", not "Updated documentation")
+- Use the imperative mood ("Fix import", not "Fixes import")
+- Limit first line to 72 characters
+
+For further guidelines, see [this
+gist](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53).
+
+### Python styleguide
+
+The recommended styleguide is
+[PEP8](https://www.python.org/dev/peps/pep-0008/), the Python standard. Using
+the source code linter [flake8](http://flake8.pycqa.org/en/latest/) will
+automatically scan your source code for compliance with PEP8.
+
+Below are some highlights of PEP8, but refer to the standard itself for full
+details.
+
+- Use 4 spaces per indentation level.
+
+- Use spaces, not tabs.
+
+- Limit lines to 79 characters at maximum.
```

Comment:
I have the line length for flake8 set to 100 in setup.cfg, but we can change that if you want.

Reply:
I have no strong opinion, 79 is just the usual prescription. I'll up the guideline to 100 if no one has any objections.